modules/planning/tasks/deciders/path_reference_decider/path_reference_decider.cc

```diff
@@ -91,6 +91,12 @@ Status PathReferenceDecider::Process(Frame *frame,
       frame->learning_based_data().learning_data_adc_future_trajectory_points();
   ADEBUG << "There are " << path_reference.size() << " path points.";
 
+  // when learning model has no output, use rule-based model instead.
+  if (path_reference.size() == 0) {
```

Comment:
shall we expose model failure(s), instead of hiding them?

Reply:
Fixed in the next pr. Thank you.